scalableminds/webknossos
# update wk-wrap, play-sbt-plugin

Third and for now final batch of dependency upgrades. This time mostly webknossos-wrap where in turn I updated and removed some dependencies in v1.1.10, compare https://github.com/scalableminds/webknossos-wrap/pull/65

Now only those backend dependency security alerts that are blocked still remain:
```
bcprov-jdk15on-1.60.jar           MEDIUM    (Waiting for new Silhouette release newer than 6.0.0)
nimbus-jose-jwt-4.41.1            CRITICAL  (Waiting for new Silhouette release newer than 6.0.0)
play-openid_2.12-2.7.2            HIGH      (Waiting for new Silhouette release newer than 6.0.0)
grpc-netty-shaded-1.37.0          MEDIUM    (waiting for new ScalaPB (protobuf/grpc) release newer than 0.11.3)
scalapb-runtime-grpc_2.12-0.11.3  CRITICAL  (waiting for new ScalaPB (protobuf/grpc) release newer than 0.11.3)
shaded-asynchttpclient-2.1.3      HIGH      (Waiting for new Play release newer than 2.8.8)
```


### URL of deployed dev instance (used for testing):
- https://updatewrap.webknossos.xyz

### Steps to test:
- load data, trace some (incl volume), save, reload
- should not be significantly slower
